ansible/roles/openshift_setup/tasks/main.yml

```diff
@@ -80,14 +89,122 @@
       oc_cluster_up_first_run: True
     when: "not 'cluster was started' in oc_cluster_status.stdout"
 
-  # Intent of this oc cluster up is generate the master-config.yaml so we can make edits to it
   - name: Run oc cluster up
     shell: "{{ oc_cmd }} cluster up --routing-suffix={{ openshift_hostname }} --public-hostname={{ openshift_hostname }} --host-pv-dir={{ persistedvol_mount_point }} --version=latest --image=docker.io/ansibleplaybookbundle/origin"
-    when: oc_cluster_up_first_run and not master_config_stat.stat.exists
+    when: (oc_cluster_up_first_run and not master_config_stat.stat.exists) or ssl_changed
```

Comment:
ssl_changed is never initialized False so it's either undefined or True. On reruns it fails.

You can do and/or `ssl_changed is defined`